tools/theme/components/badge.ts

```diff
@@ -0,0 +1,49 @@
+import { ComponentSingleStyleConfig } from '@chakra-ui/react'; // Single component style config
+
+// Badge style config
+const Badge: ComponentSingleStyleConfig = {
+  // Base style
+  baseStyle: {
+    display: 'flex',
+    alignItems: 'center',
+    borderRadius: '23px',
+    textTransform: 'inherit',
+    fontWeight: 'normal',
+    fontSize: '16px',
+    height: '32px',
+    paddingX: '1rem',
+    paddingY: '4px',
+  },
+
+  // Variants (completed, inProgress, pending, rejected, expiring, expired, active, inactive)
+  variants: {
+    completed: () => {
+      return {
+        bg: '#EEEEEE',
+      };
+    },
+    inProgress: () => ({
+      bg: '#D9F1F8',
```

Comment:
Is it ok to specify the colors like this? They don't exist in `tools/theme/colors.ts`

Reply:
In my opinion I think it would be better to add these colours to that `theme/colors.ts, especially since some of them are used multiple times throughout this file

Reply:
^ agree with @angeladietz here. We should add a `badge` property to the `colors` object that contains all of these colours, which we can reference here.

Reply:
Done! Looks much cleaner  🧹